app/src/main/java/org/supla/android/listview/ChannelListView.java

```diff
@@ -448,24 +466,12 @@ public boolean onTouchEvent(MotionEvent ev) {
 
                     if (channelLayout != null) {
                         if (channelLayout.getDetailSliderEnabled()) {
-                            Object obj = getItemAtPosition(pointToPosition((int) X, (int) Y));
-
-                            if (obj instanceof Cursor) {
-                                ChannelBase cbase;
-
-                                if (((ListViewCursorAdapter) getAdapter()).isGroup()) {
-                                    cbase = new ChannelGroup();
-                                } else {
-                                    cbase = new Channel();
-                                }
+                            ChannelBase cbase = getChannelBaseAtXY(X, Y);
+                            if(cbase != null && getDetailLayout(cbase) != null) {
+                                detailTouchDown = true;
+                                rightButtonSlided100p = channelLayout.Slided() == 200;
```

Comment:
Magic number - 200. Would be better to shift it up to constant and make a name for it

Reply:
The semantics of this code it left in its original form. You see it in the diff because of changed indentation, which is result of factoring out part of code to a subroutine. While I certainly do see a space for improvement here and in other parts of the app, my suggestion is to restrict code review comments to the changes that were actually introduced by the merge request.